### ISSUE TITLE
VM Time Drift

### ISSUE THREAD

**Issue by BronsonQuick** [Chassis/Chassis#443]
@paulgibbs had an issue with this and he's blogged about it here: https://byotos.com/2017/08/01/virtual-machines-and-time-drift/

He's also got a fix for it [here](https://gist.github.com/anonymous/166ef50d142579c7b0d53f91c686a68d) @rmccue what are your thoughts on adding this into Chassis to avoid this issue happening?

**Comment by rmccue** [comment]
If it consistently works, happy to add. Installing NTP could be a better solution but dunno.

**Comment by BronsonQuick** [comment]
Yeah, I'm not sure how to replicate this easily but I know it was frustrating @paulgibbs. NTP might work but once again I need to be able to replicate the issue which will take time. 

I think I might do a PR for this as it's such an edge case and this config change won't affect any other existing Chassis users.
